README.md

```diff
@@ -46,14 +46,20 @@ $ go install github.com/google/oss-rebuild/cmd/oss-rebuild@latest
 $ oss-rebuild --help
 ```
 
-To view the attestation for a given package, use the `get` command:
+To view the rebuild for a given package, use the `get` command:
 
 ```bash
 $ oss-rebuild get pypi absl-py 2.0.0
 ```
 
-The default output contains the rebuild's Dockerfile in base64-encoded form. To
-view this Dockerfile alone, we provide an option in the `--output` flag:
+The default output is just a summary. You can view the entire attestation payload using `--output=payload`:
```

Comment:
You could continue on with a description of other use-cases e.g. "To access the entire attestation payload instead of the summarized view, ..."